base_local_planner/src/costmap_model.cpp

```diff
@@ -59,13 +59,15 @@ namespace base_local_planner {
     if(!costmap_.worldToMap(position.x, position.y, cell_x, cell_y))
```

Comment:
Q: is (cell_x, cell_y) the robot center?

Reply:
yup